pkg/sources/github/github.go

```diff
@@ -1221,6 +1322,45 @@ func (s *Source) processPRComments(ctx context.Context, info repoInfo, chunksCha
 	return nil
 }
 
+func (s *Source) chunkIssues(ctx context.Context, repo, repoPath string, issues []*github.Issue, chunksChan chan *sources.Chunk) error {
+	for _, issue := range issues {
+
+		// Skip pull requests since covered by processPRs.
+		if issue.IsPullRequest() {
+			continue
+		}
```

Comment:
Is this something we can filter out in the `Issues.ListByRepo` call?

Reply:
Surprisingly, it doesn't seem possible. Here's the official API docs:

https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues

My guess is GH's GraphQL endpoint would let us do this, but I'm not sure.